### PR TITLE
Added my blog

### PR DESCRIPTION
**Pull request by lostfocus**

### CODE REVIEW

File: 2024.html [hunk 1]
```diff
@@ -69,6 +69,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://knowler.dev/">Nathan Knowler</a></li>
             <li><a href="https://robin.is/">Robin Massart</a></li>
             <li>Jens Oliver Meiert: <a href="https://frontenddogma.com/">Frontend Dogma</a></li>
+            <li><a href="https://lostfocus.de/">Dominik Schwind's LostFocus</a></li>
```
Comment (j9t) [inline 2024.html:72]: Minor (we might change this later, however): What do you think of the format as in the line above, indicating and sorting by your name, then linking “LostFocus”?

**Comment by j9t** [comment]
Hi @lostfocus—pardon delayed feedback: Do you mind resolving the conflict?

File: 2024.html (hunk 1 above)
Comment (lostfocus) [reply 2024.html:82]: Okay!

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
Welcome to CSS Naked Day 2024!

**Comment by lostfocus** [comment]
Thanks!

**Pull request merged**